tart2ms/tart2ms.py

```diff
@@ -355,21 +377,52 @@ def __twelveball(direction):
     for d in directions:
         LOGGER.info(f"    shapshot direction {d[0]}, {d[1]} {SkyCoord(d[0]*u.rad, d[1]*u.rad, frame='icrs').to_string('dms')}")
 
+    use_special_fn = None
     if phase_center_policy == "instantaneous-zenith":
         pass
+    elif isinstance(phase_center_policy, SkyCoord):
+        direction = np.deg2rad(np.array([[phase_center_policy.icrs.ra.value, 
+                                          phase_center_policy.icrs.dec.value]]).reshape(1, 2, 1))
     elif (phase_center_policy == "no-rephase-obs-midpoint") or \
          (phase_center_policy == "rephase-obs-midpoint"):
         direction = direction[:, :, direction.shape[2]//2].reshape(1, 2, 1)  # observation midpoint
-    elif phase_center_policy == "rephase-SCP":
-        direction = np.array([0, np.deg2rad(-90)]).reshape(1, 2, 1)
-    elif phase_center_policy == "rephase-NCP":
-        direction = np.array([0, np.deg2rad(90)]).reshape(1, 2, 1)
+    elif phase_center_policy.find("rephase-") == 0:
```

Comment:
This can be a twelveball (for which we yield a SkyCoord) or a special object or known catalog / provided position